wks-recruiter/src/main/java/pl/lodz/p/it/wks/wksrecruiter/services/TestServiceImpl.java

```diff
@@ -31,6 +33,33 @@ public TestServiceImpl(TestsRepository testsRepository, PositionsRepository posi
         this.positionsRepository = positionsRepository;
     }
 
+    @Override
+    public Test createTest(Test test) throws WKSRecruiterException {
+        try {
+            test.setActive(Boolean.TRUE);
+            test.setQuestions(new ArrayList<>());
+            testsRepository.save(test);
+            return test;
+        } catch (DuplicateKeyException exc) {
+            throw WKSRecruiterException.createException("NAME_NOT_UNIQUE",
```

Comment:
DuplicateKeyException nie zawsze dotyczy klucza głównego dokumentu. Może to też być spowodowane duplikatem klucza w _positions_